.github/workflows/ci-integration.yml

```diff
@@ -27,7 +27,6 @@ jobs:
       matrix:
         os: ['ubuntu']
         python-version:
-          - "3.10"
           - "3.11"
           - "3.12"
 
```

Comment:
Do we also want to add 3.13 here?

Reply:
We can! I believe the `openfe` stack doesn't support 3.13 yet (due to `ambertools` IIRC), so it might just fail systematically.